soc/riscv/riscv-privilege/telink_b91/Kconfig.defconfig.series

```diff
@@ -53,4 +53,7 @@ config TEST_EXTRA_STACK_SIZE
 config HAS_FLASH_LOAD_OFFSET
 	default y if BOOTLOADER_MCUBOOT
 
+config LEGACY_INCLUDE_PATH
+	default y
```

Comment:
please remove that

Reply:
if do so for now our platform builds will fail. It's due to dependent PR in hal_telink where we have legacy paths.
dependent PR was created before removing this feature in main zephyr. so it's result of long PR process.
i'll remove it in next PR after fixing paths in hal_telink

Reply:
please open a PR fixing the issues instead, this can't go in.

Reply:
FYI, you may find this useful https://github.com/zephyrproject-rtos/zephyr/blob/main/scripts/utils/migrate_includes.py